<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="to3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="from1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>from</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tip">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="from2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>from</string>
    </property>
   </widget>
   <widget class="QPushButton" name="from3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>from</string>
    </property>
   </widget>
   <widget class="QLabel" name="hint3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="newGame">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>30</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New game</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pic3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>121</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vol1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vol2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vol3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hint2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hint1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="shouldGet">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>0</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="desVolume">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>492</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBuckets">
    <property name="title">
     <string>Buckets</string>
    </property>
   </widget>
   <addaction name="menuBuckets"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
